.github/workflows/ci.yml

```diff
@@ -0,0 +1,226 @@
+---
+name: "CI"
+on:  # yamllint disable
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Linting: black"
+        run: "poetry run invoke black"
+  flake8:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Linting: flake8"
+        run: "poetry run invoke flake8"
+  hadolint:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Setup environment"
+        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+      - name: "Install hadolint"
+        run: "curl -Lo /usr/local/bin/hadolint https://github.com/hadolint/hadolint/releases/download/v2.0.0/hadolint-Linux-x86_64 && chmod +x /usr/local/bin/hadolint"
```

Comment:
Not blocking for this PR but hadolint 2.0.0 dates back to March, and the project is up to 2.7.0 now. We should plan to upgrade at some point.

Reply:
Agree. Perhaps we can upgrade at this point. If we don't do it now, I think we will stay on the 2.0.0 for a while :)

Reply:
Updated the version